plasmapy/utils/decorators/converter.py

```diff
@@ -11,6 +11,28 @@
 
 
 def angular_freq_to_hz(fn):
+    """
+    Converts angular frequency (ω) frenquency to Hertz(hz)
+
+    Parameters
+    ----------
+
+    fn : obj
+        The object to be converted
+
+    Raises
+    ------
+
+    ValueError
+        fn.__name__ can not be `to_hz`
+
+    Returns
+    -------
+
+    _result
+        result in Hertz(hz)
+
```

Comment:
We need to add some examples here.  It would be nice if they would work with doctests too.  The following might work, but you'd have to play with it

```python
import astropy.units as u
from plasmapy.utils.decorators.converter import angular_freq_to_hz

@angular_freq_to_hz
def foo(x):
    return x

foo(5 * u.rad / u.s)
<Quantity 0.79577472 Hz>
```

Reply:
Added the doctest as requested, but im not sure if what i did was enough.

Should i add another cases to this tests?